src/scripts/insertMockData.ts

```diff
@@ -1,52 +1,64 @@
 // tslint:disable no-console
-import * as uuid from 'uuid/v4';
 import { connection } from '../database/connection';
 import { NotablePerson } from '../database/entities/notablePerson';
 import { User } from '../database/entities/user';
-import { NotablePersonEvent } from '../database/entities/event';
+import { NotablePersonEvent, EventType } from '../database/entities/event';
 import { NotablePersonEventComment } from '../database/entities/comment';
-import { NotablePersonLabel } from '../database/entities/label';
-import * as Chance from 'chance';
-import { times, kebabCase } from 'lodash';
+import { NotablePersonLabel } from '../database/entities/notablePersonLabel';
+import { EventLabel } from '../database/entities/eventLabel';
+import * as faker from 'faker';
+import { times, take, uniqBy } from 'lodash';
 import { isUsingProductionDatabase } from '../env';
 
 if (isUsingProductionDatabase === false) {
-  const chance = new Chance(process.env.SEED || 1);
+  faker.seed(Number(process.env.SEED) || 1);
   connection
+    // tslint:disable-next-line:max-func-body-length
     .then(async db =>
+      // tslint:disable-next-line:max-func-body-length
```

Comment:
`tslint` being weird again?

Reply:
Looks like it does not like long functions. Ideally we should break this one into smaller functions to reduce code complexity.

Reply:
Yeah, but I was asking about why you had nested directives. I guess it's because you're using `disable-next-line` instead of simply `/* tslint:disable:rule */`

Reply:
Yeah, I used VS Code suggested "fix" which disables the rule for the next line.